codeworld-base/src/Internal/Num.hs

```diff
@@ -77,7 +77,7 @@ module Internal.Num (
 
 import qualified "base" Prelude as P
 import "base" Prelude (Bool(..), (.), (==), map)
-import Numeric
+import Numeric hiding (sqrt, pi, exp, log, logBase, sin, tan, cos, asin, atan, acos)
```

Comment:
This looks like it will break compatibility with older `base` versions.  Is it safe, or does it need an ifdef guard?